apps/backend/core/configuration/validation/validators/__init__.py

```
@@ -0,0 +1,46 @@
+    "RedisValidator",
+    # AI
+    "OpenAIValidator",
+    # Real-time
+    "PusherValidator",
+    # Authentication
+    "ClerkValidator",
+    "SupabaseValidator",
+    # Platform
+    "RobloxValidator",
+    "SentryValidator",
+]
```

Ruff (0.14.4)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)